resources/helm/testing.yaml

```diff
@@ -1766,3 +1766,93 @@ ui-bundle:
       memory: 512Mi
     requests:
       memory: 256Mi
+mod-consortia-keycloak:
+  extraEnvVars:
+  - name: SECRET_STORE_TYPE
```

Comment:
this is already a part of the eureka-common secret

Reply:
Adjusted accordingly to your suggestion.
Thanks!